app/routes/overview/components/PublicFrontpage.js

```diff
@@ -182,12 +182,13 @@ class PublicFrontpage extends Component<Props, State> {
             </li>
             <li>
               <a href="http://www.ntnu.no/studier/mtkom" target="blank">
-                <i className="fa fa-caret-right" /> Kommunikasjonsteknologi
+                <i className="fa fa-caret-right" /> Kommunikasjonsteknologi og
+                digital sikkerhet
               </a>
               <div className={styles.linkDescription}>
                 Vi bruker stadig mer av livene våre på nett, på jobb som i
-                fritid. Kommunikasjonsteknologi brukes etter hvert av alle og
-                overalt.
+                fritid. Kommunikasjonsteknologi og digital sikkerhet brukes
+                etter hvert av alle og overalt.
```

Comment:
Hmm, I feel like we should change this a bit. It doesn't quite sound right :thinking:

Reply:
Ok, changed it to "Kommunikasjonsteknologi og digital sikkerhet blir stadig viktigere i en digital verden."